### DISCUSSION TITLE
BorgBackup directly to remote location.

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#2596] [Questions] [answered]
### Is your feature request related to a problem? Please describe. 
No

### Describe the solution you'd like
BorgBackup option directly via SFTP to remote location.

### Describe alternatives you've considered
Saving borg direclty to drive and then sycning it remote location. Though it take space on primary vps.

### Additional context
N/A

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one#are-remote-borg-backups-supported
